### PR TITLE
[AGW] [New_Test_Case] Added test cases for combined attach after combined TAU reject

### PR DESCRIPTION
**Pull request by VinashakAnkitAman**
## Title
[AGW] [New_Test_Case] Added test cases for combined attach after combined TAU reject

## Summary
These test cases follow the steps:
1) Combined Attach
2) UE Context Release
3) Combined TAU
4) Reattach after TAU Reject

The MME crash was observed by Ulas for this scenario in in some cloud instance. The scripts still need to be validated against the exact scenario from cloud instance as the MME crash was not observed during the testing.

## Test Plan
Verified with Sanity and individual test case executions

Signed-off-by: VinashakAnkitAman <ankit.aman@radisys.com>

### CODE REVIEW

**Comment by 119Vik** [comment]
RestartCWAGTests

File: lte/gateway/python/integ_tests/s1aptests/test_attach_active_tau_with_combined_tala_update_reattach.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_TAU_REJECT_IND.value
+        )
+        print(
+            "************************* Received Tracking Area Update Reject "
+            "Indication"
+        )
+
+        # TAU request was sent by the UE in idle mode. So, UE state should
```
Comment (ssanadhya) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_active_tau_with_combined_tala_update_reattach.py:105]: Will the UE be set into Idle mode or detached by MME here?

File: lte/gateway/python/integ_tests/s1aptests/test_attach_inactive_tau_with_combined_tala_update_reattach.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_TAU_REJECT_IND.value
+        )
+        print(
+            "************************* Received Tracking Area Update Reject "
+            "Indication"
+        )
+
+        # TAU request was sent by the UE in idle mode. So, UE state should
+        # again be set to idle after handling TAU message. Waiting for UE
```
Comment (ssanadhya) [inline lte/gateway/python/integ_tests/s1aptests/test_attach_inactive_tau_with_combined_tala_update_reattach.py:106]: Will UE be in Idle state or implicitly detached by MME?

**Comment by VinashakAnkitAman** [comment]
The PR is closed here and raised again as PR #4421 to re-validate for some CI failures like "ci/circleci: feg-precommit"

**Pull request closed without merging**